### PR TITLE
release-stable-minor.sh: QoL improvements

### PR DESCRIPTION
**Pull request by petr-muller**
- Use the iteration with git to execute stabilization-changes.py until
  it stops making changes.
- Improve logging: mention EUSness, hide stabilization-changes.py
  log unless it fails, and log just individual iterations

Example output from 30a5c082abf96e2751c337f474d8f901c2a62640

```console
$ git co 30a5c082abf96e2751c337f474d8f901c2a62640
$ git co improve-release-scripts hack/release-stable-minor.sh
$ git commit -a -m "This needs to be clean git state"
$ ./hack/release-stable-minor.sh 4.13
4.13 is not an EUS release, will not create an EUS channel
Running stabilization-changes.py: iteration 1
Running stabilization-changes.py: iteration 2
Running stabilization-changes.py: iteration 3
Running stabilization-changes.py: iteration 4
See /tmp/tmp.D5bo4N466Z for stabilization-changes.py output if needed`
```

### CODE REVIEW

File: hack/release-stable-minor.sh [hunk 1]
```diff
@@ -34,15 +40,18 @@ fi
 
 if test "$((MINOR % 2))" -eq 0
 then
+	echo "${MAJOR_MINOR} is an EUS release, will create an EUS channel"
 	# 4.even get extended update support: https://access.redhat.com/support/policy/updates/openshift#ocp4_phases
 	CHANNELS="$(printf '%s\n%s' "${CHANNELS}" eus)"
+else
+	echo "${MAJOR_MINOR} is not an EUS release, will not create an EUS channel"
```
Comment (wking) [inline hack/release-stable-minor.sh:47]: "will not create an EUS channel" -> "no EUS channel to update"?  This command is `sed`ing to adjust `filters` in channel files that already exist.

File: hack/release-stable-minor.sh [hunk 2]
```diff
@@ -34,15 +40,18 @@ fi
 
 if test "$((MINOR % 2))" -eq 0
 then
+	echo "${MAJOR_MINOR} is an EUS release, will create an EUS channel"
 	# 4.even get extended update support: https://access.redhat.com/support/policy/updates/openshift#ocp4_phases
```
Comment (wking) [inline hack/release-stable-minor.sh:44]: nit: keep this "what does `MINOR % 2` have todo with EUS?" up at the top of this block, or shift it to the front of the `if` statement or something?

File: hack/release-stable-minor.sh [hunk 3]
```diff
@@ -52,4 +61,24 @@ done
...
+touch "${STABILIZATION_LOG}"
+
+# Execute stabilization-changes.py until it stops making changes
+while true; do
+	echo "Running stabilization-changes.py: iteration ${I}" | tee -a "${STABILIZATION_LOG}"
+	if ! "${DIR}"/stabilization-changes.py &>> "${STABILIZATION_LOG}"; then
+			echo "FAIL: stabilization-changes.py output tail: (see full log in ${STABILIZATION_LOG})"
+			tail "${STABILIZATION_LOG}"
+			break
+  fi
```
Comment (wking) [inline hack/release-stable-minor.sh:75]: nit: space vs. tab indent?

File: hack/release-stable-minor.sh [hunk 4]
```diff
@@ -52,4 +61,24 @@ done
...
-exec "${DIR}/stabilization-changes.py"
+I=1
+STABILIZATION_LOG="$(mktemp)"
+touch "${STABILIZATION_LOG}"
+
+# Execute stabilization-changes.py until it stops making changes
+while true; do
+	echo "Running stabilization-changes.py: iteration ${I}" | tee -a "${STABILIZATION_LOG}"
+	if ! "${DIR}"/stabilization-changes.py &>> "${STABILIZATION_LOG}"; then
+			echo "FAIL: stabilization-changes.py output tail: (see full log in ${STABILIZATION_LOG})"
```
Comment (wking) [inline hack/release-stable-minor.sh:72]: nit: 3 tab indent when the parent line was only 1.  I'd expect two tabs here.

Comment (petr-muller) [reply hack/release-stable-minor.sh:72]: that's what I get for using a home computer w/o properly set up tools :D

**Review by wking — APPROVED** [review]
/lgtm

**Pull request merged**